game/src/sandbox/dashboards/risks.rs

```diff
@@ -0,0 +1,425 @@
+use std::collections::BTreeSet;
+use std::fmt::Display;
+
+use abstutil::{abbreviated_format, prettyprint_usize};
+use geom::{Duration, Polygon, Pt2D, Time};
+use map_gui::tools::ColorScale;
+use sim::{Problem, TripMode};
+use widgetry::{
+    Color, DrawWithTooltips, EventCtx, GeomBatch, GfxCtx, Outcome, Panel, State, Text, TextExt,
+    Toggle, Widget,
+};
+
+use crate::app::{App, Transition};
+use crate::common::color_for_mode;
+use crate::sandbox::dashboards::DashTab;
+
+pub struct RiskSummaries {
+    panel: Panel,
+}
+
+impl RiskSummaries {
+    pub fn new(ctx: &mut EventCtx, app: &App, filter: Filter) -> Box<dyn State<App>> {
+        let mut filters = Vec::new();
+        for mode in TripMode::all() {
+            filters.push(Toggle::colored_checkbox(
+                ctx,
+                mode.ongoing_verb(),
+                color_for_mode(app, mode),
+                filter.modes.contains(&mode),
+            ));
+        }
+
+        Box::new(RiskSummaries {
+            panel: Panel::new(Widget::col(vec![
+                DashTab::RiskSummaries.picker(ctx, app),
+                Widget::col(vec![
+                    "Filters".text_widget(ctx),
```

Comment:
Hmm, both risks only affect cyclists. I guess we have plans to add some for pedestrians, so leaving the filters in could make sense. But if the problems for pedestrians are exclusive to them and transit users, then I'd vote just removing the filter entirely. (Distinguishing problems happening to people just walking the entire trip vs only to/from a bus stop doesn't make much sense)

Reply:
Yes good point. 

I think we should get rid of the filter, and better clarify which type of agent each metric applies to and *only* include those agents in the totals - e.g. don't count car drivers in the "unaffected" row.

In that same vein, it's a little unintuitive to phrase this one in terms of cars:
<img width="317" alt="Screen Shot 2021-05-13 at 11 24 18 AM" src="https://user-images.githubusercontent.com/217057/118169413-c342a080-b3dd-11eb-911b-1a6c1be9c244.png">

I think we want it to be "cyclists whom a car wanted to overtake". It is admittedly kind of a mouthful, but it puts the focus on the safety of the cyclist rather than the convenience of the driver.

Reply:
Whoops, I guess just a thumbs up isn't a super clear indication all of this SGTM. Also, I like the simplicity of the legend! Great way to avoid having to align tick marks.

Reply:
>  I like the simplicity of the legend! Great way to avoid having to align tick marks.

<img width="337" alt="Screen Shot 2021-05-13 at 9 24 48 AM" src="https://user-images.githubusercontent.com/217057/118176903-d73ed000-b3e6-11eb-99bf-47c35d02dc00.png">

I actually started down that road, but gave up after a while... even if I did get everything aligned, it's a lot of text.